vsphere/datadog_checks/vsphere/metrics.py

```diff
@@ -403,6 +410,65 @@
     HISTORICAL: [],
 }
 
+VSAN_METRICS = {
+    'vsan.cluster.congestion',
```

Comment:
can we separate these into entity types?

Reply:
Done.